graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchList.jsx

```diff
@@ -39,6 +40,12 @@ const NoSavedSearches: StyledComponent<{}, ThemeInterface, *> = styled(Alert)`
   align-items: center;
 `;
 
+const DeleteButton: StyledComponent<{}, ThemeInterface, *> = styled.span`
```

Comment:
I see that we are using the `*` for other component type definitions in this file, but I would prefer `HTMLSpanElement`.